changelog.d/20240725_212933_codewithemad_kill.md

```diff
@@ -0,0 +1 @@
+- [Improvement] New `kill` command for local and development environments. (by @CodeWithEmad)
```

Comment:
shouldn't this be better off in documentation/troubleshooting instead of a new command? While I understand the need, I don't feel like the tutor codebase would be the right place.

Reply:
Well yes. It can be in docs too, but the point is to have a quick command to stop the environment without wasting time. Each stop can take up to 10 - 15 seconds and goes up depending on your plugins and machine.